src/cephadm/cephadm.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/python3
+# mypy: ignore-errors
```

Comment:
This ignores errors that have previously been ignored by mypy because `cephadm` (without the `.py` extension) wasn't a proper Python module.

Reply:
this has bothered me too. when we re-enable this, it would also be nice to refactor the type checking from py2 -> py3 style syntax